bootstrap/config.aws-aws.tpl

```diff
@@ -15,65 +21,18 @@
           ]
         },
         "Instance": {
-          "Plugin": "instance-aws",
-          "Properties": {
-            "RunInstancesInput": {
-              "ImageId": "{{ ref "/aws/amiid" }}",
-              "InstanceType": "{{ ref "/aws/instancetype" }}",
-              "KeyName": "{{ ref "/aws/keyname" }}",
-              "SubnetId": "{{ ref "/aws/subnetid" }}",
-              {{ if ref "/aws/instanceprofile" }}"IamInstanceProfile": {
-                "Name": "{{ ref "/aws/instanceprofile" }}"
-              },{{ end }}
-              "SecurityGroupIds": [ "{{ ref "/aws/securitygroupid" }}" ]
-            },
-            "Tags": {
-              "Name": "{{ ref "/aws/stackname" }}-manager",
-              "Deployment": "Infrakit",
-              "Role" : "manager"
-            }
-          }
+          {{ include "instace-aws.tpl" "config.aws-aws.tpl" }}
```

Comment:
to get a cleaner repo, all the included files should be in a subdirectory (include), and the include statement could be {{ include "include/instance-aws.tpl" }}